core/store/Cargo.toml

```diff
@@ -9,6 +9,7 @@ rust-version = "1.56.0"
 edition = "2021"
 
 [dependencies]
+anyhow = "1"
```

Comment:
`anyhow` is meant for "sloppy" error management which you do at the edges of the system (CLI). `store` feels like the place where we care about pedantically correct error management, so it feels weird that it has to depend on anyhow.